internal/controlplane/http.go

```diff
@@ -37,7 +38,7 @@ func (srv *Server) addHTTPMiddleware(root *mux.Router, cfg *config.Config) {
 			Str("path", r.URL.String()).
 			Msg("http-request")
 	}))
-	root.Use(handlers.RecoveryHandler())
+	root.Use(gorillahandlers.RecoveryHandler())
```

Comment:
Thought for FUP PR, we should probably just yank out the gorilla handlers depedency and carry the handler directly since we are using so few of them (I think this is the only one?). I think the project is deprecated too.